Documentation for sap-converged-cloud Plan Region Mappings (#859)
kyma-project/kyma-environment-broker: docs/user/03-10-service-description.md

| **maxSurge[<sup>2</sup>](#update)**       | int    | Specifies the maximum number of virtual machines that are created during an update.        |    No    | `4`           |
| **maxUnavailable[<sup>2</sup>](#update)** | int    | Specifies the maximum number of virtual machines that can be unavailable during an update. |    No    | `1`           |

>  SAP Converged Plan cannot be provisioned in all regions. This restriction is enforced through region mapping functionality configured by [`sapConvergedCloudPlanRegionMappings`](https://github.com/kyma-project/kyma-environment-broker/blob/48d5f55dfacfc511ead132fb77f522abc7e382e7/resources/keb/values.yaml#L215). The list enables users to set mappings from BTP region, passed to provisioning endpoints in HTTP path param (list key), to Kyma regions (list entries). Based on that configuration and passed path param, broker schema is populated with values only from given btp region.

Reviewer: ```suggestion
 The SAP Converged Cloud plan cannot be provisioned in all regions. This restriction is enforced through the region mapping functionality configured by [`sapConvergedCloudPlanRegionMappings`](https://github.com/kyma-project/kyma-environment-broker/blob/48d5f55dfacfc511ead132fb77f522abc7e382e7/resources/keb/values.yaml#L215). The list enables you to set mappings from a BTP region that are passed to provisioning endpoints in the HTTP path parameter (list key) and to Kyma regions (list entries). Based on that configuration and the passed path parameter, the broker schema is populated only with values from the given btp region.
```

>  The SAP Converged Cloud plan cannot be provisioned in all SKR regions. This restriction is enforced through the region mapping functionality configured by [`sapConvergedCloudPlanRegionMappings`](https://github.com/kyma-project/kyma-environment-broker/blob/48d5f55dfacfc511ead132fb77f522abc7e382e7/resources/keb/values.yaml#L215). The list enables you to map BTP region that is passed to the provision endpoint in the HTTP path parameter (map key) to Kyma regions (list entries). Based on that configuration and the passed path parameter, the broker schema is populated only with values from the mapped list. In case of empty mapping configuration or passing provisioning path parameter that does not contain configured region then `sap-converged-cloud` plan is not rendered in the schema.

Reviewer: ```suggestion
 The SAP Converged Cloud plan cannot be provisioned in all SKR regions. This restriction is enforced through the region mapping functionality configured by [`sapConvergedCloudPlanRegionMappings`](https://github.com/kyma-project/kyma-environment-broker/blob/48d5f55dfacfc511ead132fb77f522abc7e382e7/resources/keb/values.yaml#L215). The list enables you to map a BTP region, which is passed to the provisioning endpoint in an HTTP path parameter (map key), to Kyma regions (list entries). Based on that configuration and the passed path parameter, the broker schema is populated only with values from the mapped list. In case of an empty mapping configuration or passing a provisioning path parameter that does not contain the configured region, the `sap-converged-cloud` plan is not rendered in the schema.
```
